hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/ChunkUtils.java

```diff
@@ -163,14 +161,27 @@ private static long writeDataToFile(File file, ChunkBuffer data,
 
   private static long writeDataToChannel(FileChannel channel, ChunkBuffer data,
       long offset) {
+    FileLock lock = null;
     try {
+      lock = channel.lock(offset, data.remaining(), false);
```

Comment:
We may use try-with-resource as below.  Then, we can remove the releaseLock(..) method.
```
  private static long writeDataToChannel(FileChannel channel, ChunkBuffer data,
      long offset) {
    try(FileLock lock = channel.lock(offset, data.remaining(), false)) {
      channel.position(offset);
      return data.writeTo(channel);
    } catch (IOException e) {
      throw new UncheckedIOException(e);
    }
  }
```